apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -214,6 +321,11 @@ function SkeletonSchedule() {
                     </Box>
                 ))
             }
+
+            <CustomEventsBox customEvents={skeletonSchedule.customEvents} scheduleNames={[]} isSkeletonMode={true} />
```

Comment:
`scheduleNames={[]}` is a magic array here; skeletonSchedule doesn't have a `scheduleNames` prop, but `CustomEventDetailView` is disabled in skeleton mode so the lack of this prop doesn't matter.